build/Makefile

```diff
@@ -23,9 +23,9 @@ SCX_INTERMEDIATE_DIR := $(SCX_DIR)/intermediate/$(BUILD_CONFIGURATION)
 DSC_TARGET_DIR := $(DSC_DIR)/release
 
 RUBY_DIR := $(BASE_DIR)/source/ext/ruby
+RUBY_VER := 2.3.0
```

Comment:
Created a pull request in the Build-OMS-Agent-for-Linux project with the documentation update: [PR 19](https://github.com/Microsoft/Build-OMS-Agent-for-Linux/pull/19)
Note: autoconf does not need to be updated to successfully build omsagent.